shared/util/quit-helper.desktop.js

```diff
@@ -47,7 +56,8 @@ function _executeActionsFromRenderer (actions: Array<Action>) {
   ipcRenderer.send('executeActions', actions)
 }
 
-export function executeActions (actions: Array<Action>) {
+export function executeActionsForContext (context: Context) {
```

Comment:
This extra composability doesn't give us anything right now. Everything has
to call these extra functions for no benefit. If we need this complexity
later we can add it but if this all this thing needs to do it doesn't make
the code simpler or clearer

On Friday, July 15, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/util/quit-helper.desktop.js
> <https://github.com/keybase/client/pull/3469#discussion_r71046088>:
>
> > @@ -47,7 +56,8 @@ function _executeActionsFromRenderer (actions: Array<Action>) {
> >    ipcRenderer.send('executeActions', actions)
> >  }
> >
> > -export function executeActions (actions: Array<Action>) {
> > +export function executeActionsForContext (context: Context) {
>
> I'm okay having a helper that does executeActionsForContext but I think
> we should keep executeActions since it's a smaller piece that composes
> better.
>
> Then executeActionsForContext is just a composition of those two functions.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/keybase/client/pull/3469/files/696a85227ca0f7720ffeeea0c9eb4a149973b035#r71046088>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAfhUzekY1xugJal9rWHtf7EggEXoifjks5qWAE-gaJpZM4JMsjQ>
> .
>